src/modules/data/Transaction.ts

```diff
@@ -42,12 +42,12 @@ export class Transaction
     /**
      * The array of references to the unspent output of the previous transaction
      */
-    public inputs: TxInputs[];
+    public inputs: TxInput[];
```

Comment:
```suggestion
    public inputs: Input[];
```
https://github.com/bpfkorea/agora/blob/cb259d978d5e9240a46be2a309c5e08370477cd9/source/agora/consensus/data/Transaction.d#L52

Is there a reason why you don't use the same name as Agora?

Reply:
It will avoid colliding with other packages of the symbal.